### PR TITLE
Adding feature for intertechno

### PR DESCRIPTION
**Pull request by tarioch**
Signed-off-by: Patrick Ruckstuhl patrick@ch.tario.org (github: tarioch)

### CODE REVIEW

File: features/openhab-addons-external/pom.xml [hunk 1]
```diff
@@ -36,6 +36,7 @@
                                 <artifact><file>src/main/resources/conf/caldavio.cfg</file><type>cfg</type><classifier>caldavio</classifier></artifact>
                                 <artifact><file>src/main/resources/conf/caldavPersonal.cfg</file><type>cfg</type><classifier>caldavPersonal</classifier></artifact>
                                 <artifact><file>src/main/resources/conf/comfoair.cfg</file><type>cfg</type><classifier>comfoair</classifier></artifact>
+								<artifact><file>src/main/resources/conf/culintertechno.cfg</file><type>cfg</type><classifier>culintertechno</classifier></artifact>
```
Comment (kaikreuzer) [inline features/openhab-addons-external/pom.xml]: could you please replace the tabs by spaces here, so that it is inline with the rest of the file?

**Comment by kaikreuzer** [comment]
Thanks, great to know that this is working :+1:

**Comment by tarioch** [comment]
Fixed spacing, sorry about that.

**Comment by kaikreuzer** [comment]
No worries, thanks for the PR!

**Pull request merged**